roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1163,6 +1167,49 @@ def init_local_facts(self, facts=None):
         self.changed = changed
         return new_local_facts
 
+    def init_in_docker_facts(self, facts):
+        facts['is_containerized'] = os.path.isfile('/run/ostree-booted')
+        deployment_type = facts['common']['deployment_type']
+
+        # TODO: Most of these shouldn't actually make it into the facts
+        docker = dict()
+        if deployment_type in ['enterprise','openshift-enterprise']:
+            docker['master_image_name'] = 'openshift3/ose'
+            docker['ovs_image_name'] = 'openshift3/openvswitch'
+            docker['node_image_name'] = 'openshift3/node'
+            docker['etcd_image_name'] = 'rhel7/etcd'
+        elif deployment_type == 'atomic-enterprise':
+            docker['master_image_name'] = 'aep3/aep'
+            docker['ovs_image_name'] = 'aep3/openvswitch'
+            docker['node_image_name'] = 'aep3/node'
+            docker['etcd_image_name'] = 'rhel7/etcd'
+        else:
+            docker['master_image_name'] = 'openshift/origin'
+            docker['ovs_image_name'] = 'openshift/openvswitch'
+            docker['node_image_name'] = 'openshift/node'
+            docker['etcd_image_name'] = 'rhel7/etcd'
```

Comment:
@ibotty I believe that sdodson put that in place until the centos equivalent is published. I could be mistaken though.